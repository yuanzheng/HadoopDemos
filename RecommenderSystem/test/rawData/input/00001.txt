1,1001,5.0
1,1012,3.5
2,1002,4.0