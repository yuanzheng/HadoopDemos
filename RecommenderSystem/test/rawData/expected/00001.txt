1	1012:3.5,1001:5.0
2	1002:4.0